### PR TITLE
PV Inverter hide phases without values

### PR DESCRIPTION
**Pull request by mr-manuel**

### CODE REVIEW

File: pages/settings/devicelist/ac-in/PageAcInModelDefault.qml [hunk 1]
```diff
@@ -79,6 +79,8 @@ ObjectModel {
 					id: phasePower
 					source: root.bindPrefix + "/Ac/L" + (model.index + 1) + "/Power"
 				}
+
+				visible: phasePower.valid
```
Comment (blammit) [inline pages/settings/devicelist/ac-in/PageAcInModelDefault.qml:83]: If you rebase against `main`, this should now be `phasePower.isValid` instead

File: pages/settings/devicelist/ac-in/PageAcInModelDefault.qml [hunk 2]
```diff
@@ -113,6 +115,7 @@ ObjectModel {
 				text: qsTrId("ac-in-modeldefault_energy_x").arg(model.index + 1)
 				dataSource: "%1/Ac/L%2/Energy/Forward".arg(root.bindPrefix).arg(model.index + 1)
 				unit: VenusOS.Units_Energy_KiloWattHour
+				visible: dataValid
```
Comment (blammit) [inline pages/settings/devicelist/ac-in/PageAcInModelDefault.qml:118]: Should now be `visible: dataItem.isValid`

**Review by blammit — CHANGES_REQUESTED** [review]
f92438ea834e65a516bc5b317e231add3896559a can be removed from the PR

**Comment by mr-manuel** [comment]
See https://github.com/victronenergy/gui-v2/pull/617#discussion_r1450149142. Shoud stay as it is.

**Pull request closed without merging**